README.md

```diff
@@ -62,28 +62,34 @@ The easiest way to compile the code and run the system locally is using [Docker]
 
 Don't forget to run the docker daemon!
 
-## Build the container
-
-```terminal
-$ cd opencbdc-tx                 # change to the project directory
-$ sudo -s                        # open a root shell (needed for docker)
-# docker build . -t opencbdc-tx  # build the container
-```
-
 ## Launch the System
 
 **Note:** You will need to both run the system and interact with it; you can either use two shells, or you can add the `--detach` flag when launching the system (note that it will then remain running till you stop it, e.g., with `docker stop`).
 Additionally, you can start the atomizer architecture by passing `--file docker-compose-atomizer.yml` instead.
 
+_The commands below will build a new image every time that you run it.
+You can remove the `--build` flag after the image has been built to avoid rebuilding.
+To run the system with our pre-built image proceed to the [next section](#launch-the-system-with-a-pre-built-image) for the commands to run._
+
 1. Run the System
    ```terminal
-   # docker-compose --file docker-compose-2pc.yml up
+   # docker compose --file docker-compose-2pc.yml up --build
    ```
 1. Launch a container in which to run wallet commands (use `--network atomizer-network` instead of `--network 2pc-network` if using the atomizer architecture)
    ```terminal
    # docker run --network 2pc-network -ti opencbdc-tx /bin/bash
    ```
 
+## Launch the System With a Pre-built Image
+
+We publish new docker images for all commits to `trunk`.
+You can find the images [in the Github Container Registry](https://github.com/mit-dci/opencbdc-tx/pkgs/container/opencbdc-tx).
+
+**Note:** You must use `docker compose` instead of `docker-compose` for this approach to work or you will need to pull the docker image `docker pull ghcr.io/mit-dci/opencbdc-tx`.
```

Comment:
```suggestion
**Note:** You must use `docker compose` (not `docker-compose`) for this approach to work or you will need to pull the image manually with `docker pull ghcr.io/mit-dci/opencbdc-tx`.
```

Reply:
Great call! I really appreciate the suggestions to improve documentation!